### ISSUE TITLE
[Bug]: Storage is not released after DROP DATABASE

### ISSUE THREAD

**Issue by domingozhang** [matrixorigin/matrixone#6871]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Before dropping, the size of directory is 91GB 
After dropping about 60GB imported data with CSV, the size of directory is still 91GB

**Comment by XuPeng-SH** [comment]
GC will be implemented in 0.7
#6893

**Comment by XuPeng-SH** [comment]
implemented in 0.7

**Comment by XuPeng-SH** [comment]
duplicated with #6893
